services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.db.models.functions import TruncDate
+
+import init_django_orm  # noqa: F401
```

Comment:
?

Reply:
I didn't know about lookups at that time and had to google other comparison methods